test/lib/runtime/broker_test.exs

```diff
@@ -3838,7 +3838,8 @@ defmodule Ask.BrokerTest do
     [survey, group, _, _, _] = create_running_survey_with_channel_and_respondent()
 
     phone_number = "1-734-555-1212"
-    respondent = insert(:respondent, survey: survey, respondent_group: group, phone_number: phone_number, sanitized_phone_number: Ask.Respondent.sanitize_phone_number(phone_number))
+    canonical_phone_number = Ask.Respondent.canonicalize_phone_number(phone_number)
+    respondent = insert(:respondent, survey: survey, respondent_group: group, phone_number: phone_number, sanitized_phone_number: canonical_phone_number, canonical_phone_number: canonical_phone_number)
```

Comment:
the factory can't be used set the sanitized and canonical number from the given phone_number?

Reply:
yes, hadn't think about it, good idea! ✨ 
I will investigate a little bit more to know how exmachina works and how to capture the arguments passed in the insert, and add a commit with this

Reply:
@bcardiff  Apparently [Allow more control over factory definitions #333](https://github.com/thoughtbot/ex_machina/pull/333) was merged in [ex_machina 2.3.0](https://github.com/thoughtbot/ex_machina/blob/master/CHANGELOG.md#230) and surveda is currently using 1.0.2 😞 
It's a nice-to-have new feature

With 1.0.2, the result of setting a `phone_number` in de `insert(:respondent)` function but not the `canononical_phone_number`/`sanitized_phone_number` is a respondent with inconsistent data